screens/NotificationScreen.js

```diff
@@ -1,16 +1,102 @@
-import React from "react";
-import { StyleSheet, Text, View } from "react-native";
+import React, { useContext, useEffect, useState } from "react";
+import { Text, View } from "react-native";
+import axios from "axios";
 
-import Notification from "../components/Notification";
+import AuthContext from "../auth/Context";
+import { apiConfig } from "../config/config";
+import { HOST_WITH_PORT } from "../environment";
+import FixedText from "../components/FixedText";
+import Question from "../components/Question";
 
 export default function NotificationScreen({ navigation, route }) {
-  const { body, title } = route.params;
+  const authContext = useContext(AuthContext);
+  const [data, setData] = useState();
+  const { notification } = route.params;
 
-  return (
-    <View>
-      <Notification body={body} title={title} />
-    </View>
-  );
-}
+  const contentType =
+    notification.notification.request.content.data.content_type;
+  const itemId = notification.notification.request.content.data.item_id;
+
+  useEffect(() => {
+    getContentInfo();
+  }, []);
+
+  const getContentInfo = async () => {
+    if (contentType === "Static") {
+      const contentInfo = await axios
+        .get(HOST_WITH_PORT + `${apiConfig.urls.static}/${itemId}/`, {
+          auth: apiConfig.auth,
+        })
+        .catch((err) => {
+          console.log(err);
+        });
+      setData(contentInfo);
+    } else if (contentType === "Question") {
+      const contentInfo = await axios
+        .get(HOST_WITH_PORT + `${apiConfig.urls.question}/${itemId}/`, {
+          auth: apiConfig.auth,
+        })
+        .catch((err) => {
+          console.log(err);
+        });
+      setData(contentInfo);
+    }
+  };
+
+  //Posts the answer to the API
+  const postData = async (payLoad) => {
+    axios
+      .post(HOST_WITH_PORT + "answers/", payLoad, {
+        auth: apiConfig.auth,
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  };
 
-const styles = StyleSheet.create({});
+  const answerCallBack = (answer) => {
+    //will need to change these values later to fit specific users and questions
+    let payLoad = { user: authContext.user.id, question: itemId };
```

Comment:
Is this comment relevant? I think you've taken care of the changes it's mentioning.